sample-code/src/foundation-models/azure-openai.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AzureOpenAiChatClient,
+  AzureOpenAiChatCompletionResponse,
+  AzureOpenAiEmbeddingClient,
+  AzureOpenAiEmbeddingResponse
+} from '@sap-ai-sdk/foundation-models';
+import { createLogger } from '@sap-cloud-sdk/util';
+
+const logger = createLogger({
+  package: 'sample-code',
+  messageContext: 'foundation-models-azure-openai'
+});
+
+/**
+ * Ask Azure OpenAI gpt-35-turbo model about the capital of France.
```

Comment:
[pp] Maybe not have the model name in the comments. May lead to inconsistency in the future if we change the parameter and forget the comment.